src/main/java/uk/gov/hmcts/reform/ccd/documentam/service/ValidationUtils.java

```diff
@@ -61,15 +58,15 @@ public static void validateInputParams(String pattern, String... inputString) {
         }
     }
 
-    public static void validateLists(List<?>... inputList) {
+    public void inputLists(List<?>... inputList) {
```

Comment:
that comment seems to be misunderstood. I had meant:
`public static void validateLists(List<?>... inputList)`
should be
`public static void validateLists(List<?>... inputLists)`